### ISSUE TITLE
Still get LNK2001 errormessages (unresolved ... JNI_CreateJavaVM) after uninstalling Java8 SDK 64bit

### ISSUE THREAD

**Issue by RichardBollinger** [joeferner/node-java#288]
This same issue is present in the google group. I started there not knowing how difficult it would be to resolve. I tried following the instructions but ran into all kinds of problems. I installed and reinstalled software that was required and uninstalled software that caused problems. Some of the versions mentioned are not the latest versions. But, I cannot be sure that is a problem.

I presume that Visual Studio is called upon to compile C++ code. And, that this is done without, or behind the scenes, creating a VS project. Searches for similar problems with C++ projects seemed to have been resolved by putting jvm.lib explicitly in the path of the project. But, that cannot be done with a non-existing or out-of-view VS project.

So, I tried putting jvm.lib anywhere I could think of where MSBUILD or VCBUILD would trip over it and find their missing JavaVM token.

I have created this issue here because I cannot any entry in the problem that might reach a solution.

I am at your mercy. If any more information is needed beyond what I attached to the posts on google groups, then just let me know and I will fetch it.

Thanks,
Rick

**Comment by rfreed23** [comment]
FYI - Had the same problem, and resolved it by installing a 32-bit Java. Apparently the issue was that java couldn't get to anything above the 32-bit line.  This ensured that all of the elements/resources are 'below the line' and accessible.

**Comment by RichardBollinger** [comment]
I am past this problem now. Choosing the option 1 to avoid installing Visual Studio seems not to work. Installing VS 2013 seems to be the magical combination. It comes with .NET Framework 4.5.1. So, I set msvs_version=2013 and I could npm install java ok. I installed it in the base node directory. And, I installed it in my Eclipse project. I do not trust the global installation in Windows 7. It puts in under AppData/Roaming or something like that under my user documents.
